package.json

```diff
@@ -58,23 +58,23 @@
   "dependencies": {
     "@ciscospark/common": "1.43.2",
     "@ciscospark/helper-html": "1.43.2",
-    "@ciscospark/internal-plugin-conversation": "1.43.2",
-    "@ciscospark/internal-plugin-encryption": "1.43.2",
-    "@ciscospark/internal-plugin-feature": "1.43.2",
-    "@ciscospark/internal-plugin-flag": "1.43.2",
-    "@ciscospark/internal-plugin-mercury": "1.43.2",
-    "@ciscospark/internal-plugin-metrics": "1.43.2",
-    "@ciscospark/internal-plugin-presence": "1.43.2",
-    "@ciscospark/internal-plugin-search": "1.43.2",
-    "@ciscospark/internal-plugin-team": "1.43.2",
-    "@ciscospark/internal-plugin-user": "1.43.2",
-    "@ciscospark/plugin-authorization": "1.43.2",
-    "@ciscospark/plugin-logger": "1.43.2",
-    "@ciscospark/plugin-people": "1.43.2",
-    "@ciscospark/plugin-phone": "1.43.2",
-    "@ciscospark/spark-core": "1.43.2",
+    "@ciscospark/internal-plugin-conversation": "1.47.3",
+    "@ciscospark/internal-plugin-encryption": "1.47.3",
+    "@ciscospark/internal-plugin-feature": "1.47.3",
+    "@ciscospark/internal-plugin-flag": "1.47.3",
+    "@ciscospark/internal-plugin-mercury": "1.47.3",
+    "@ciscospark/internal-plugin-metrics": "1.47.3",
+    "@ciscospark/internal-plugin-presence": "1.47.3",
+    "@ciscospark/internal-plugin-search": "1.47.3",
+    "@ciscospark/internal-plugin-team": "1.47.3",
+    "@ciscospark/internal-plugin-user": "1.47.3",
+    "@ciscospark/plugin-authorization": "1.47.3",
+    "@ciscospark/plugin-logger": "1.47.3",
+    "@ciscospark/plugin-people": "1.47.3",
+    "@ciscospark/plugin-phone": "1.47.3",
+    "@ciscospark/spark-core": "1.47.3",
     "@ciscospark/storage-adapter-local-forage": "1.40.1",
-    "@ciscospark/storage-adapter-local-storage": "1.43.2",
+    "@ciscospark/storage-adapter-local-storage": "1.47.3",
```

Comment:
can we split the version update to a different commit for the changelog please and thanks?